vochain/scrutinizer/process.go

```diff
@@ -153,20 +153,20 @@ func (s *Scrutinizer) ProcessList(entityID []byte,
 	return procs, err
 }
 
-// ProcessCount return the number of processes indexed
-func (s *Scrutinizer) ProcessCount(entityID []byte) int64 {
+// ProcessCount returns the number of processes indexed
+func (s *Scrutinizer) ProcessCount(entityID []byte) uint64 {
```

Comment:
We should try to unify integer types, I think for ProcessCount() uint32 should mot more than enough

Reply:
should I change the CountStore Count type to be uint32? This would include processes, entitites, envelopes, txs

Reply:
You already changed it from int64 to uint64, no?

Reply:
yes, for processes. But now processes, entities, envelopes, and tx counts will all share the same type- CountStore. Do we want those all to be uint32?

Reply:
Better not make changes not related with this commit, so lets just change the minimum values. However uint32 makes sense in my opinion here